### DISCUSSION TITLE
Wifi status monitor

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15045] [Ideas and suggestions]
You want to see the status of your drone, but there's no status monitor. what do you do? nothing, it's not possible. you can't view the status of drones. why? because.

**Comment by Ek-Videogames** [comment]
It could be implemented similarly to how 'sonar transceivers' work for the navigation terminal, some kind of remote status monitor connection.

**Reply by girlyguppy** [reply]
yeah
or maybe you just link it to a shuttle if lazy

it would be cool if we could set a certain hud layout as default, so we could have many status monitors for different drones, without the player each time having to try to drag around the windows
